_posts/2020-04-08-release-2.3.0.md

```diff
@@ -11,8 +11,8 @@ As of 08/04 2020 the new version 2.3.0 is out
 ## Standardization & Conformance Improvements
 
 - [#1222]({{ site.github.issues_url }}/1222) Spark version checking for Conformance and Standardization (currently supported versions >=2.4.2, <3.0.0)
-- [#1157]({{ site.github.issues_url }}/1260) The group explosion strategy for mapping conformance rules is turned on by default. 
-- [#1202]({{ site.github.issues_url }}/1202) Plugin that pushes control info to Kafka (https://github.com/AbsaOSS/enceladus-plugin-kafka) is integrated as a built-in plugin in Enceladus. Thanks @TarannumPraveen . 
+- [#1157]({{ site.github.issues_url }}/1157) The group explosion strategy for mapping conformance rules is turned on by default. 
+- [#1202]({{ site.github.issues_url }}/1202) [Plugin that pushes control info to Kafka](https://github.com/AbsaOSS/enceladus-plugin-kafka) is integrated as a built-in plugin in Enceladus. Thanks @TarannumPraveen . 
```

Comment:
```suggestion
- [#1202]({{ site.github.issues_url }}/1202) [Plugin](https://github.com/AbsaOSS/enceladus-plugin-kafka) that pushes control info to Kafka is integrated as a built-in plugin in Enceladus. Thanks @TarannumPraveen . 
```

Reply:
Such a link I would expect to link to a definition of a plugin. Not a specific one. 🤔

Reply:
How about now? I do understand your concern, I will leave it up to you in the end

Reply:
Used The instead of A.